### PR TITLE
Task with Collections

### PR DESCRIPTION
**Pull request by antonbl32**

### CODE REVIEW

File: src/main/java/by/antonsh/lesson10/Ex29.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+заполнить рандом. Удалить неудовлетворительные оценки
+ */
+public class Ex29 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> grade=new ArrayList<Integer>(
+                random.ints(15,3,10).boxed().collect(Collectors.toList()));
+        System.out.println(grade);
+        grade= (ArrayList<Integer>) grade.stream().filter(a->a>=7).collect(Collectors.toList());
+        System.out.println(grade);
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson10/Ex29.java:20]: Please, sometimes format code to right tabulation. Ctrl+Alt+L. Also in new ArrayList<Integer> write <Integer> not necessary, only new ArrayList<>. Diamond syntax

File: src/main/java/by/antonsh/lesson10/Ex29.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.stream.Collectors;
+
+/*
+Создать список оценок учеников с помощью ArrayList,
+заполнить рандом. Удалить неудовлетворительные оценки
+ */
+public class Ex29 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> grade=new ArrayList<Integer>(
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson10/Ex29.java]: work at the interface level: List<Integer> grade = new ArrayList

File: src/main/java/by/antonsh/lesson10/Ex30.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Создать рандомную коллекцию чисел, удалить повторяющиеся
+ */
+public class Ex30 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> list=new ArrayList<Integer>(
+                random.ints(15,3,10).boxed().collect(Collectors.toList()));
+        System.out.println(list);
+        list= (ArrayList<Integer>) list.stream().distinct().collect(Collectors.toList());
+        System.out.println(list);
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson10/Ex30.java:18]: it's also ok, but for variant, you can set List<> in Set<>

File: src/main/java/by/antonsh/lesson10/Ex30.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import java.util.Random;
+import java.util.stream.Collectors;
+
+/*
+Создать рандомную коллекцию чисел, удалить повторяющиеся
+ */
+public class Ex30 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> list=new ArrayList<Integer>(
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson10/Ex30.java]: interface level

File: src/main/java/by/antonsh/lesson10/Ex31.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.stream.Collectors;
+
+/*
+Создать список оценок учеников с помощью ArrayList, заполнить случайными оценками.
+Найти самую высокую оценку с использованием итератора.
+ */
+public class Ex31 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> grade=new ArrayList<Integer>(
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson10/Ex31.java]: interface level

File: src/main/java/by/antonsh/lesson10/Ex32.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+Имеется текст, создать для него частотный словарь
+ */
+public class Ex32 {
+    public static void main(String[] args) {
+        String text = " Коси траву, коси траву, пока солнце высоко.".toLowerCase();
+        HashMap<String, Integer> map = new HashMap<>();
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson10/Ex32.java]: interface level

File: src/main/java/by/antonsh/lesson10/Ex29.java (hunk 1 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson10/Ex29.java:20]: Ok,thks

File: src/main/java/by/antonsh/lesson10/Ex29.java (hunk 2 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson10/Ex29.java]: Ok,thks

File: src/main/java/by/antonsh/lesson10/Ex30.java (hunk 4 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson10/Ex30.java]: Changed

File: src/main/java/by/antonsh/lesson10/Ex30.java (hunk 3 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson10/Ex30.java:18]: I know, thks

File: src/main/java/by/antonsh/lesson10/Ex31.java (hunk 5 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson10/Ex31.java]: fixed

**Review by svetamolchanova — APPROVED** [review]

**Pull request merged**